### PR TITLE
Update uk translation of basics (to 1.2.1) and enum (to 1.5.0) sections

### PR DESCRIPTION
**Pull request by fly1ngDream**

### CODE REVIEW

File: uk/lessons/basics/enum.md [hunk 1]
```diff
@@ -195,11 +195,20 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
...
 
 ```elixir
-iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)
+iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
 [1, 2, 3]
 ```
+
+### uniq_by
+
+`uniq_by/2` також видаляє дуюлікати з перерахувань, але дозволяє нам вказати функцію для того, щоб здійснити унікальне порівняння.
```
Comment (filalex77) [inline uk/lessons/basics/enum.md]:
```suggestion
`uniq_by/2` також видаляє дублікати з перерахувань, але дозволяє нам вказати функцію для того, щоб здійснити унікальне порівняння.
```

File: uk/lessons/basics/enum.md [hunk 2]
```diff
@@ -195,11 +195,20 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
 [%{count: 1}, %{count: 4}]
 ```
 
-### uniq_by
+## uniq
```
Comment (lex111) [inline uk/lessons/basics/enum.md]:
```suggestion
### uniq
```

Comment (fly1ngDream) [reply uk/lessons/basics/enum.md]: Thanks for your suggestion (typo)

**Comment by lex111** [comment]
@filalex77 this PR is good for merge?

**Comment by filalex77** [comment]
@lex111 I'd say yes.

**Comment by lex111** [comment]
@fly1ngDream @filalex77 спасибі вам обом!

**Pull request merged**